gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -347,6 +353,8 @@ public GoogleCloudStorageImpl(
               this.storageOptions, this.backgroundTasksThreadPool, credentials);
     }
     this.downscopedAccessTokenFn = downscopedAccessTokenFn;
+
+    this.ugiUserName = getUgiUserName();
```

Comment:
Do we need to call this if AuditLog is disabled?

Reply:
Added check.